### PR TITLE
Allow user to access uploadAsset function from db

### PR DESCRIPTION
**Pull request by cheungpat**
connects #429

### CODE REVIEW

File: packages/skygear-core/lib/database.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ export class Database {
    */
   _presaveSingleValue(value) {
     if (isAsset(value) && value.file) {
-      return makeUploadAssetRequest(this.container, value);
+      return this.uploadAsset(this.container, value);
```
Comment (carmenlau) [inline packages/skygear-core/lib/database.js]: `uploadAsset` only accept asset, `this.container` is not needed

Comment (carmenlau) [inline packages/skygear-core/lib/database.js]: `uploadAsset` only accept asset, `this.container` is not needed

**Comment by cheungpat** [comment]
@carmenlau fixed

**Comment by carmenlau** [comment]
Merged via 2972049

**Pull request closed without merging**